src/ImportPage.css

```diff
@@ -21,12 +21,35 @@
   transition: 0.2s;
 }
 
-/* React logo Spinner */
-.reactlogo {
-  height: 40vmin;
-  pointer-events: none;
+kbd {
+  display: inline-block;
+  padding: 0 0.5em;
+  border: 1px solid #d7dfe4;
+  margin: 0 0.2em;
+  background-color: #f6f6f6;
+  border-radius: 0.2em;
+}
+
+/* Landing Graphic */
+.browserScreenshot {
+  width: 35rem;
+  max-width: inherit;
+  align-self: center;
+  justify-content: center;
+  border: 1px solid #d7dfe4;
+  border-radius: 0.4em;
+  box-shadow: 0 2px 4px #ddd;
+}
+
+.legendKey {
+  font-size: calc(8px + 1.5vmin);
+}
+
+.legendKey > svg {
+  padding-left: 2rem;
 }
 
+/* Old React spinner animation */
```

Comment:
Could we remove this dead code? Unfortunately it's not easy to locate dead CSS so imo it'll be better to remove them. If we need these styles in the future we can always come back to this PR